src/components/checkbox/_checkbox.scss

```diff
@@ -179,3 +182,162 @@
     }
   }
 }
+
+@mixin checkbox--experimental {
+  // Spacing between checkboxes
+  .#{$prefix}--form-item.#{$prefix}--checkbox-wrapper {
+    margin-bottom: rem(6px);
+  }
+
+  // Spacing above collection of checkboxes
+  .#{$prefix}--form-item.#{$prefix}--checkbox-wrapper:first-of-type {
+    margin-top: rem(3px);
+  }
+
+  // Spacing below collection of checkboxes
+  .#{$prefix}--form-item.#{$prefix}--checkbox-wrapper:last-of-type {
+    margin-bottom: rem(3px);
+  }
+
+  // Visually, we'll hide the checkbox input and create our own styled version
+  // to match the specs.
+  .#{$prefix}--checkbox {
+    @include hidden;
+  }
+
+  // The label corresponds to the content inside of the `label` tag. Since we're
+  // creating our own checkbox style, we'll need to position this in order to
+  // accomodate the spacing needed for the checkbox.
+  .#{$prefix}--checkbox-label {
+    @include reset;
+    @include font-family;
+    @include font-smoothing;
+    //@include type-scale-item('b');
+
+    position: relative;
+    display: flex;
+    cursor: pointer;
+    padding-left: rem(24px);
+    min-height: rem(24px);
+    user-select: none;
+  }
+
+  // Spacing for presentational checkbox
+  .#{$prefix}--checkbox-label::before {
+    box-sizing: border-box;
+    content: '';
+
+    // According to the spec, we'll want the bounding box for our checkbox to
+    // be 16px. The border size will be what will be updated during the
+    // different checkbox states.
+    width: rem(12px);
+    height: rem(12px);
+    margin: rem(2px);
+
+    // We need to position the pseudo-element absolutely in the space that we've
+    // created with the padding from the label itself. We position only with
+    // `top` since we don't want the checkbox to be centered vertically with the
+    // text overflows.
+    position: absolute;
+    left: 0;
+    top: rem(2px);
+    background-color: $ui-02;
+    border: 1px solid $ui-05;
+    border-radius: 1px;
+  }
+
+  // Create the appearance of the check in the `after` pseudo-element
+  .#{$prefix}--checkbox-label::after {
+    content: '';
+    position: absolute;
+    left: rem(5px);
+    top: rem(7px);
+    width: rem(6px);
+    height: rem(3px);
+    background: none;
+    border-left: 1px solid $ui-02;
+    border-bottom: 1px solid $ui-02;
+    transform: scale(0) rotate(-45deg);
+    transform-origin: bottom right;
+    margin-top: rem(-3px);
+  }
+
+  //----------------------------------------------
+  // Hover
+  // ---------------------------------------------
+
+  // Hover color for border
+  .#{$prefix}--checkbox-wrapper:hover .#{$prefix}--checkbox-label::before {
+    border-color: ç$ibm-color__gray-70;
```

Comment:
Seems like a strange character before the variable, not sure what it is or if it's from the diff view: `ç$ibm-color__gray-70;`